.azure/azure-pipelines.ci.yml

```diff
@@ -509,46 +510,121 @@ stages:
+      randomLossDenominator: 1000
+      rttMs: (5,50,200)
+      logProfile: Datapath.Light
+  - template: ./templates/run-wanperf.yml
+    parameters:
+      image: windows-2022
+      jobName: wanperf_5Mbps_1_percent_loss
+      displayName: 5 Mbps (1% Loss)
+      iterations: 3
+      durationMs: 10000
+      rateMbps: 5
+      bottleneckQueueRatio: (0.25,1.0,5.0)
+      randomLossDenominator: 100
+      rttMs: (5,50,200)
+      logProfile: Datapath.Light
+  - template: ./templates/run-wanperf.yml
+    parameters:
+      image: windows-2022
+      jobName: wanperf_30Mbps_no_loss
+      displayName: 30 Mbps (No Loss)
+      iterations: 3
+      durationMs: 10000
+      rateMbps: 30
+      bottleneckQueueRatio: (0.25,1.0,5.0)
+      randomLossDenominator: 0
+      rttMs: (5,50,200)
+      logProfile: Datapath.Light
+  - template: ./templates/run-wanperf.yml
+    parameters:
+      image: windows-2022
+      jobName: wanperf_30Mbps_0_1_percent_loss
+      displayName: 30 Mbps (0.1% Loss)
+      iterations: 3
+      durationMs: 10000
+      rateMbps: 30
+      bottleneckQueueRatio: (0.25,1.0,5.0)
+      randomLossDenominator: 1000
       rttMs: (5,50,200)
       logProfile: Datapath.Light
   - template: ./templates/run-wanperf.yml
     parameters:
       image: windows-2022
-      jobName: wanperf_30Mbps
-      displayName: 30 Mbps
+      jobName: wanperf_30Mbps_1_percent_loss
+      displayName: 30 Mbps (1% Loss)
       iterations: 3
       durationMs: 10000
       rateMbps: 30
-      bottleneckQueueRatio: (0.25,1.0)
+      bottleneckQueueRatio: (0.25,1.0,5.0)
+      randomLossDenominator: 100
       rttMs: (5,50,200)
       logProfile: Datapath.Light
   - template: ./templates/run-wanperf.yml
     parameters:
       image: windows-2022
-      jobName: wanperf_100Mbps
-      displayName: 100 Mbps
+      jobName: wanperf_100Mbps_no_loss
+      displayName: 100 Mbps (No Loss)
       iterations: 3
       durationMs: 10000
       rateMbps: 100
-      bottleneckQueueRatio: (0.25,1.0)
+      bottleneckQueueRatio: (0.25,1.0,5.0)
+      randomLossDenominator: 0
+      rttMs: (5,50,200)
+      logProfile: Datapath.Light
+  - template: ./templates/run-wanperf.yml
+    parameters:
+      image: windows-2022
+      jobName: wanperf_100Mbps_0_1_percent_loss
+      displayName: 100 Mbps (0.1% Loss)
+      iterations: 3
+      durationMs: 10000
+      rateMbps: 100
+      bottleneckQueueRatio: (0.25,1.0,5.0)
+      randomLossDenominator: 1000
+      rttMs: (5,50,200)
+      logProfile: Datapath.Light
+  - template: ./templates/run-wanperf.yml
+    parameters:
+      image: windows-2022
+      jobName: wanperf_100Mbps_1_percent_loss
+      displayName: 100 Mbps (1% Loss)
+      iterations: 3
+      durationMs: 10000
+      rateMbps: 100
+      bottleneckQueueRatio: (0.25,1.0,5.0)
+      randomLossDenominator: 100
       rttMs: (5,50,200)
       logProfile: Datapath.Light
 
-- ${{ if not(eq(variables['Build.Reason'], 'Schedule')) }}:
```

Comment:
Not necessary.

Reply:
Changes for this file have been reverted.